parachains/runtimes/assets/asset-hub-polkadot/src/weights/pallet_xcm.rs

```diff
@@ -48,240 +47,240 @@ use core::marker::PhantomData;
+	/// Storage: `ParachainSystem::HostConfiguration` (r:1 w:0)
+	/// Proof: `ParachainSystem::HostConfiguration` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainSystem::PendingUpwardMessages` (r:1 w:1)
+	/// Proof: `ParachainSystem::PendingUpwardMessages` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	fn send() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `109`
-		//  Estimated: `3574`
-		// Minimum execution time: 29_223_000 picoseconds.
-		Weight::from_parts(29_633_000, 0)
-			.saturating_add(Weight::from_parts(0, 3574))
+		//  Measured:  `75`
+		//  Estimated: `3540`
+		// Minimum execution time: 27_822_000 picoseconds.
+		Weight::from_parts(28_198_000, 0)
+			.saturating_add(Weight::from_parts(0, 3540))
 			.saturating_add(T::DbWeight::get().reads(5))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
-	/// Storage: ParachainInfo ParachainId (r:1 w:0)
-	/// Proof: ParachainInfo ParachainId (max_values: Some(1), max_size: Some(4), added: 499, mode: MaxEncodedLen)
+	/// Storage: `ParachainInfo::ParachainId` (r:1 w:0)
+	/// Proof: `ParachainInfo::ParachainId` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
 	fn teleport_assets() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `1489`
-		// Minimum execution time: 24_692_000 picoseconds.
-		Weight::from_parts(25_226_000, 0)
+		// Minimum execution time: 24_216_000 picoseconds.
+		Weight::from_parts(24_534_000, 0)
 			.saturating_add(Weight::from_parts(0, 1489))
 			.saturating_add(T::DbWeight::get().reads(1))
 	}
-	/// Storage: ParachainInfo ParachainId (r:1 w:0)
-	/// Proof: ParachainInfo ParachainId (max_values: Some(1), max_size: Some(4), added: 499, mode: MaxEncodedLen)
+	/// Storage: `ParachainInfo::ParachainId` (r:1 w:0)
+	/// Proof: `ParachainInfo::ParachainId` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
 	fn reserve_transfer_assets() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `1489`
-		// Minimum execution time: 19_046_000 picoseconds.
-		Weight::from_parts(19_397_000, 0)
+		// Minimum execution time: 18_483_000 picoseconds.
+		Weight::from_parts(18_703_000, 0)
 			.saturating_add(Weight::from_parts(0, 1489))
 			.saturating_add(T::DbWeight::get().reads(1))
 	}
-	/// Storage: Benchmark Override (r:0 w:0)
-	/// Proof Skipped: Benchmark Override (max_values: None, max_size: None, mode: Measured)
+	/// Storage: `Benchmark::Override` (r:0 w:0)
+	/// Proof: `Benchmark::Override` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn execute() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 9_714_000 picoseconds.
-		Weight::from_parts(9_898_000, 0)
+		// Minimum execution time: 18_446_744_073_709_551_000 picoseconds.
+		Weight::from_parts(18_446_744_073_709_551_000, 0)
```

Comment:
we're stopping this from happening... switched to Max weight?

Reply:
`pallet_xcm::execute` is disabled on BridgeHubs, in master/release there are empty runtimes,
more accurate weights will come here: https://github.com/paritytech/cumulus/pull/2528

Reply:
(prior version of kusama-asset-hub already had this change rolled out)

Reply:
Given that we're not keen on people executing arbitrary xcm on system parachains at the moment, I think this seems the right setting.